test/plugin_functional/plugins/elasticsearch_client_plugin/server/plugin.ts

```diff
@@ -26,22 +26,22 @@ export class ElasticsearchClientPlugin implements Plugin {
       { path: '/api/elasticsearch_client_plugin/context/ping', validate: false },
       async (context, req, res) => {
         const { body } = await context.core.elasticsearch.client.asInternalUser.ping();
-        return res.ok({ body });
+        return res.ok({ body: JSON.stringify(body) });
```

Comment:
Why it's necessary? It's already a string

Reply:
The new client has `body` a `Boolean`, and `HttpResponsePayload` doesn't allow booleans per https://github.com/elastic/kibana/blob/feceb0f98eb817f065834f8b6c9c628cee41383a/src/core/server/http/router/response.ts#L81.